app/assets/main/react_components/User/Card.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+
+function Card({ monthlyAction, categoryColor }) {
```

Comment:
Change this to arrow function to keep code uniform :) 

`const Card = ({monthlyAction, categoryColor}) => { ... }`